lib/ReactViews/Analytics/PointParameterEditor.jsx

```diff
@@ -20,61 +18,45 @@ const PointParameterEditor = React.createClass({
         viewState: React.PropTypes.object
     },
 
+    componentWillMount() {
+        if (!defined(this.pointParameterEditorCore)) {
+            this.pointParameterEditorCore = new PointParameterEditorCore(this.props.previewed,
```

Comment:
This won't work.  If the component receives new props, the `Core` component won't get them.  You can implement more React lifecycle methods, but there's probably a more Reacty way to do this.  Let's talk about it.